### DISCUSSION TITLE
Welcome to PSDocs.Azure Discussions!

### DISCUSSION THREAD

**Discussion by ElinoraWhite** [Azure/PSDocs.Azure#13] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself.

**Comment by hristozkov69** [comment]
Hi Bernie, I was trying to use the modules for readme generation for some bicep files, but  I hit something like the bug described here:
https://github.com/Azure/PSDocs.Azure/issues/35
I am using a script to gather all the needed parameters for  the command Invoke-PSDocument and the command is almost the same as the one you provided in the readme section:

 Get-AzDocTemplateFile -Path $tempPath | ForEach-Object {
    # Generate a standard name of the markdown file. i.e. <name>_<version>.md
    $template = Get-Item -Path $_.TemplateFile;
    # Generate markdown
    Invoke-PSDocument -Module PSDocs.Azure -OutputPath $templateDir -InputObject $template.FullName -InstanceName $docName -Verbose -Debug

  }
However the output of it is:

Line |
 100 |      Invoke-PSDocument -Module PSDocs.Azure -OutputPath $templateDir - …
     |      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Cannot bind argument to parameter 'Name' because it is null.

Any ideas why this can be happening.
